### ISSUE TITLE
3D models not shown in simple app. Only 2D drawing visible.

### ISSUE THREAD

**Issue by AliFlux** [helix-toolkit/helix-toolkit#1017]
## TL;DR

I made a simple app using Helix/SharpDX, with some basic configuration. The 2D elements (title, rectangles, rotation cue) are shown, but the 3D elements in the view (grid, cube, coordinate system) are all invisible.

## Details

Following screenshot describes the anomaly:

![shot](https://user-images.githubusercontent.com/8333794/49371281-c7e8d200-f718-11e8-9cf2-144ddafa74e2.PNG)

Here is the **code**:

```cs
<hx:Viewport3DX
    x:Name="view1"
    Margin="10"
    Title="Hello World"
    BackgroundColor="White"
    EnableDesignModeRendering="True"
    FXAALevel="Low"
    MSAA="Maximum"
    ShowViewCube="True"
    ZoomExtentsWhenLoaded="True"
    ShowCoordinateSystem="True">
    <hx:AmbientLight3D Color="Blue" />

</hx:Viewport3DX>
```

```cs
public ManView()
{
    InitializeComponent();

    var camera = new HelixToolkit.Wpf.SharpDX.PerspectiveCamera
    {
        Position = new Point3D(3, 3, 5),
        LookDirection = new Vector3D(-3, -3, -5),
        UpDirection = new Vector3D(0, 1, 0),
        FarPlaneDistance = 5000000
    };

    view1.Camera = camera;
    view1.EffectsManager = new EffectsManager();
    view1.ModelUpDirection = new Vector3D(0, 1, 0);
    
    var grid = new LineGeometryModel3D();

    grid.Geometry = LineBuilder.GenerateGrid(new Vector3(0, 1, 0), -5, 5, -5, 5);
    grid.Color = Colors.Black;
    grid.Transform = new TranslateTransform3D(0, -3, 0);

    view1.Items.Add(grid);
}
```

Any idea what could possibly be the reason? The **demos in the github source are all perfectly working**. I'm using `HelixToolkit.Wpf.SharpDX` `2.5.1` from nuget. My system config are:

Intel i7 4700HQ
Geforce 760M
Windows 10 Creators Update

Here is the project zip for testing: [Man3D.zip](https://drive.google.com/open?id=10M0KeX8KAoWLzLAWlZCmrptqzrFOxjo8)

**Comment by AliFlux** [comment]
Thanks! That really fixed it.
